rubato/utils/vector.py

```diff
@@ -490,18 +491,27 @@ def __pow__(self, other: any) -> Vector:
         if isinstance(other, Vector):
             return Vector(self.x**other.x, self.y**other.y)
 
+    def __ipow__(self, other: any) -> Vector:
+        return self.__pow__(other)
```

Comment:
for functions like these that just use other methods, use `__ipow__ = __pow__` instead of declaring a new function